packages/xchain-binance/src/client.ts

```diff
@@ -238,19 +225,8 @@ class Client implements BinanceClient, XChainClient {
    *
    * @throws {Error} Thrown if phrase has not been set before. A phrase is needed to create a wallet and to derive an address from it.
    */
-  getAddress = (): string => {
-    if (!this.address) {
-      const address = crypto.getAddressFromPrivateKey(this.getPrivateKey(), getPrefix(this.network))
-      if (!address) {
-        throw new Error(
-          'Address has to be set. Or set a phrase by calling `setPhrase` before to use an address of an imported key.',
-        )
-      }
-
-      this.address = address
-    }
-    return this.address
-  }
+  getAddress = (index = 0): string =>
```

Comment:
Change interface of `getAddress` to  `getAddress(index?: number)` (line 67)

Reply:
done